src/optimum/rbln/modeling_base.py

```diff
@@ -416,7 +416,20 @@ def __call__(self, *args, **kwargs):
         return self.forward(*args, **kwargs)
 
     def __repr__(self):
-        return repr(self.model) + repr(self.rbln_submodules)
+        has_submodule = len(self.rbln_submodules) > 0
+        repr_str: str = f"<{self.__class__.__name__}>\n"
+        repr_str += f"- Total {len(self.model)} Runtimes"
+        repr_str += f" and {len(self.rbln_submodules)} Submodules\n" if has_submodule else "\n"
+        repr_str += "[Runtimes]\n"
+        repr_str += "\n".join([repr(model) for model in self.model])
+        repr_str += "\n"
+
+        if has_submodule > 0:
+            for i, submodule in enumerate(self.rbln_submodules):
+                repr_str += f"[Submodules {i} : {self._rbln_submodules[i]['name']}]\n"
```

Comment:
아래와 같이 변경하면, 동작에 이슈가 있을까요?
```suggestion
            for i, submodule in enumerate(self._rbln_submodules):
                repr_str += f"[Submodules {i} : {submodule["name"]}]\n"
```

Reply:
submodule의 repr도 가져와야해서(바로 아랫줄), 이러면 동작에 영향이 갈 거 같습니다.